wiki/Beatmapping/Circle_Size/en.md

```diff
@@ -0,0 +1,32 @@
+---
+tags:
+- CS
+---
+
+# Circle Size
+
+Circle Size, commonly abbreviated as *CS*, is a difficulty setting which determines the size of [hit objects](/wiki/Hit_Objects) on [beatmaps](/wiki/Beatmaps). The values range from 2 to 7; higher circle size values will make hit objects smaller. Using the highest value will make hit objects as big as the cursor. Most beatmaps use a value between 3 and 5, while values between 5 and 6 are sometimes used and values of 6 and above are almost never used. High circle size values are likely to be seen when using the [Hard Rock](/wiki/HR) mod as it increases circle size. Conversely, the [Easy](/wiki/HR) mod will lower circle size.
```

Comment:
> Using the highest value will make hit objects as big as the cursor.

Would be incorrect if the cursor size is set other than 1X (or automatic cursor size was enabled)... not sure how technical we want to be. The suggestion I wrote for this line switches "big" with "small", but I just wanted to point this out, I guess.

Reply:
agree, this seems a bit incorrect. this was pulled straight from another article iirc. think I'll clean it up instead